packages/tools/src/tools/segmentation/CircleROIStartEndThresholdTool.ts

```diff
@@ -881,9 +881,9 @@ class CircleROIStartEndThresholdTool extends CircleROITool {
  * target volume enclosed by the rectangle.
  *
  * @param data - The annotation tool-specific data.
- * @param targetId - The volumeId of the volume to display the stats for.
+ * @param metadata - The metadata annotation.
  */
-function defaultGetTextLines(data): string[] {
```

Comment:
can we have it as defaultGetTextLines(data, {} ), i'm pretty sure in future something else need to get injected too